mig/shared/output.py

```diff
@@ -2726,6 +2726,9 @@ def file_format(configuration, ret_val, ret_msg, out_obj):
                 file_content += line
         elif entry['object_type'] == 'binary':
             file_content = entry['data']
+        elif entry['object_type'] == 'error_text':
```

Comment:
Hm - I don't know this code but I can't quite follow how `error_text` could previously have worked is this is now necessary? Were we in fact missing appropriate handling of this rather commony used object_type?

Reply:
That was/is indeed the case and you can easily test by explicitly calling the `cat.py` backend with `output_format=file` and `path=someboguspathname` on one of the production sites. That combination will (almost) silently error out by just handing you a 0 byte file.
It should be noted that it is specific for `output_format=file`, which is not nearly as widely used as `html` or `txt`, but it should of course still be addressed.

Reply:
The motivation for not previously handling that error message in the `file_format` helper likely was that it would then have ended up inside the downloaded file rather than a message to the user.